bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/regexp/TreeBuilder.java

```diff
@@ -119,7 +119,7 @@ private RegExpTerm readRegTerm() {
         }
         return new RegExpAtomQuantifier(reAtom, quantifier);
     }
-    
+
```

Comment:
One general comment. Avoid reformatting the complete file to avoid this kind of diffs.